Source/RomanNumeral.c

```diff
@@ -6,11 +6,21 @@
 #include <stdio.h>
 #include "RomanNumeral.h"
 
-static char * Append(char *output, const char *toAppend)
+static char *Append(char *output, const char *toAppend)
 {
    return output + sprintf(output, "%s", toAppend);
 }
 
 void RomanNumeral_Convert(unsigned input, char *output)
 {
+   memset(output, 0, 1000);
```

Comment:
I'm 99% sure that `Append` will do this for you via `sprintf`.

Reply:
Which would in general be a better solution since you don't know for sure that `output` is actually 1000 bytes long in the general case.